libraries/botbuilder-azure/src/cosmosDbStorage.ts

```diff
@@ -261,7 +261,7 @@ function getOrCreateCollection(client: DocumentClient, databaseLink: string, col
             value: string;
         }[];
     } = {
-        query: 'SELECT * FROM root r WHERE r.id=@id',
+        query: 'SELECT r._self FROM root r WHERE r.id=@id',
```

Comment:
Line 235 in "getOrCreateDatabase" needs to be fixed as well.

Reply:
This is done!